data/json/body_parts.json

```diff
@@ -600,6 +600,103 @@
     "bmi_encumbrance_threshold": 20,
     "bmi_encumbrance_scalar": 0.15
   },
+  {
+    "id": "gastropod_foot",
+    "type": "body_part",
+    "name": "gastropod foot",
+    "accusative": { "ctxt": "bodypart_accusative", "str": "gastropod foot" },
+    "heading": "gastropod foot",
+    "heading_multiple": "gastropod feet",
+    "hp_bar_ui_text": "G. FOOT",
+    "encumbrance_text": "Running and swimming are slowed.",
+    "encumbrance_threshold": 20,
+    "main_part": "gastropod_foot",
+    "connected_to": "torso_waist",
+    "opposite_part": "gastropod_foot",
+    "limb_types": [ [ "leg_l", 0.8 ], [ "leg_r", 0.8 ], [ "foot_l", 0.5 ], [ "foot_r", 0.5 ], [ "tail", 0.2 ] ],
+    "limb_scores": [ [ "reaction", 0.3 ] ],
+    "is_vital": true,
+    "hit_size": 4,
+    "hit_difficulty": 1.2,
+    "side": "both",
+    "grabbing_effect": "grabbing_head",
+    "stylish_bonus": 3,
+    "hot_morale_mod": 2,
+    "cold_morale_mod": 2,
+    "fire_warmth_bonus": 150,
+    "squeamish_penalty": 7,
+    "base_hp": 60,
+    "drench_capacity": 700,
+    "bionic_slots": 58,
+    "flags": [ "LIMB_LOWER" ],
```

Comment:
You could add a conditional flag for `SLUDGE_IMMUNE` and move that from the trait.

Reply:
I'm not understanding what you mean? SLUDGE_IMMUNE is a json flag so that it's not hardcoded to that trait only for resisting the field.

Reply:
What you can do with limbs is add conditional flags that only work as long as the limb is below their encumbrance threshold - if your foot is naked you get to resist sludging, but if you put on a~~giant plastic bag~~ *artisian survivor gastropod foot cover* it should probably not protect you.

Reply:
Can you link me to an example of that since this is my first real foray into limbs?  That sounds cool!

Reply:
> dd conditional flags that only work as long as the limb is below their encumbrance threshold - if your foot is naked you get to resist sludging, but if you put on a~giant plastic bag~ _artisian survivor gastropod foot cover_ it should probably not protect you.

That's not currently how the document reads and looking at the code I don't think that's in code yet either. Currently if I'm reading this correctly conditional flags only check if the limb has 1 or more HP.